modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -88,7 +88,7 @@ def put(x):
     def put(x):
         return PythonWrapper.put(x, hash=False)
 
-    def deploy(func, args):
+    def deploy(func, args=[]):
```

Comment:
let's not make a [mutable object to be a default argument](https://docs.aws.amazon.com/codeguru/detector-library/python/default-argument-mutable-objects/)
```suggestion
    def deploy(func, args=tuple()):
```
or
```suggestion
    def deploy(func, *args):
```

Reply:
*args wont work here because a list of arguments is passed when this function is called